packages/react-strict-dom/tests/__snapshots__/html-test.native.js.snap-native

```diff
@@ -1312,57 +1313,27 @@ exports[`<html.*> style polyfills default inherited text styles 1`] = `
         {
           "color": "red",
           "fontFamily": "Arial",
-          "fontSize": 32,
+          "fontSize": 20,
           "fontStyle": "italic",
           "fontVariant": "variant",
           "fontWeight": "300",
-          "lineHeight": 48,
+          "lineHeight": 30,
```

Comment:
The test's input value was changed from `1.5` to `30px`, and before the fix to the `create` unit processing this would have rendered a value of `600` instead of `30`.